cmd/migrate/cockroachdb-enterprise-operator/migrate-helm.go

```diff
@@ -0,0 +1,47 @@
+package cockroachdb_enterprise_operator
+
+import (
+	"github.com/cockroachdb/helm-charts/pkg/migrate"
+	"github.com/spf13/cobra"
+)
+
+var (
+	crdbCluster     string
+	clusterManifest string
+)
+
+var buildManifestFromHelm = &cobra.Command{
```

Comment:
Is this command specific to helm, or would it work with other statefulset-based crdb deployments? Is the idea that it works with statefulsets, but only supports options available in the helm chart?

Reply:
It is specifically for the statefulset created using helm chart of cockroachdb.